Article/Article.js

```diff
@@ -26,4 +26,8 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('.article');
+
+articles.forEach(function(item) {
+  return new articles(item);
```

Comment:
this looks like a typo? 
```suggestion
  return new Article(item);
```